cmd/create/operatorroles/cmd.go

```diff
@@ -219,12 +219,17 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
+	operatorRolePolicyPrefix, err := aws.GetOperatorRolePolicyPrefixFromCluster(cluster, r.AWSClient)
```

Comment:
Crucial change. Iff the user creates the cluster with a non standard acc role as the installer role we use the operator role prefix instead of the acc role prefix when creating the operator policies